mne/viz/topomap.py

```diff
@@ -217,7 +217,10 @@ def _plot_update_evoked_topomap(params, bools):
             tp = cont.collections[0]
             visible = tp.get_visible()
             patch_ = tp.get_clip_path()
-            color = tp.get_color()
+            try:
+                color = tp.get_edgecolors()
+            except AttributeError:  # old MPL
```

Comment:
can we mention in the comment which version is the critical one?  will make it easier to remove the try/except later as our min version changes.